examples/visualization/plot_eeglab_head_sphere.py

```diff
@@ -0,0 +1,133 @@
+"""
+.. _ex-topomap-eeglab-style:
+
+========================================
+How to plot topomaps the way EEGLAB does
+========================================
+
+If you have previous EEGLAB experience you may have noticed that topomaps
+(topoplots) generated using mne-python look a little different from those
+created in EEGLAB. If you prefer the EEGLAB style this example will show you
+how to calculate head sphere origin and radius to obtain EEGLAB-like channel
+layout in mne.
+
+"""
+# Authors: Mikołaj Magnuski <mmagnuski@swps.edu.pl>
+#
+# License: BSD (3-clause)
+import numpy as np
+from matplotlib import pyplot as plt
+
+import mne
+
+
+print(__doc__)
+
+###############################################################################
+# Create fake data
+# ----------------
+#
+# First we will create a simple evoked object with a single timepoint using
+# biosemi 10-20 channel layout.
+
+biosemi_montage = mne.channels.make_standard_montage('biosemi64')
+n_channels = len(biosemi_montage.ch_names)
+fake_info = mne.create_info(ch_names=biosemi_montage.ch_names, sfreq=250.,
+                            ch_types='eeg')
+
+rng = np.random.RandomState(0)
+data = rng.normal(size=(n_channels, 1)) * 1e-6
+fake_evoked = mne.EvokedArray(data, fake_info)
+fake_evoked.set_montage(biosemi_montage)
+
+###############################################################################
+# Calculate sphere origin and radius
+# ----------------------------------
+#
+# EEGLAB plots head outline at the level where head circumference is measured
+# in 10-20 system (a line going through Fpz, T8/T4, Oz and T7/T3 channels).
+# MNE places the head outline lower on the z dimension: approximattely at the
+# level of LPA, RPA fiducial markers. Therefore to use the EEGLAB layout we
+# have to move the origin of the reference sphere (a sphere that is used as a
+# reference when projecting channel locations to a 2d plane) a few centimeters
+# up.
+#
+# Instead of approximating this position by eye, as we did in the sensor
+# locations tutorial :ref:`tut-sensor-locations`, we will calculate it using
+# the position of Fpz, T8, Oz and T7 channels available in out montage.
```

Comment:
```suggestion
# Instead of approximating this position by eye, as we did in `the sensor
# locations tutorial <tut-sensor-locations>`, here we will calculate it using
# the position of Fpz, T8, Oz and T7 channels available in our montage.
```